.github/workflows/lint.yaml

```diff
@@ -29,13 +30,15 @@ jobs:
           key: ${{ runner.os }}-lerna-${{ hashFiles('**/yarn.lock') }}
 
       - name: Install dependencies
+        if: steps.node-modules-cache.outputs.cache-hit != 'true'
         run: |
-          yarn
+          yarn --prefer-offline
+          node_modules/.bin/lerna bootstrap
       - name: Lint & Fix
         run: |
-          git config --global user.email "noreply@hoprnet.org"
-          git config --global user.name "HOPR Linting robot"
+          git config user.name github-actions
```

Comment:
why are we doing this?

Reply:
See https://github.com/actions/checkout#push-a-commit-using-the-built-in-token - it seems that this account is using a proper SSH setup. Also git pull looks slightly differently.

Reply:
you don't need to change the email and name to change the token ? ...

Reply:
just following the documentation ... this is what https://github.com/actions/checkout#push-a-commit-using-the-built-in-token suggests

Reply:
I guess they're triggering different actions under the hood when changing the user and / or email to github-actions

Reply:
seems unlikely?

Reply:
obsolete now